ui/po/pt_BR.po

```diff
@@ -425,14 +425,14 @@ msgid ""
 "lifetime access).\n"
 "        "
 msgstr ""
-" A primeira vez que você usar o Breezy Desktop, uma conexão com a internet "
+"           A primeira vez que você usar o Breezy Desktop, uma conexão com a internet "
 "será necessária para obter a licença do seu dispositivo.\n"
-" \n"
-" Não se preocupe, não há nenhuma obrigação de doar neste momento. Se você "
+"          \n"
+"          Não se preocupe, não há nenhuma obrigação de doar neste momento. Se você "
 "ainda não o fez, será fornecida uma licença de teste para que você possa "
 "decidir se o Breezy Desktop atende às suas necessidades.\n"
-" \n"
-" Uma vez que você obtenha uma licença, de teste ou não, você poderá usar o "
+"          \n"
+"          Uma vez que você obtenha uma licença, de teste ou não, você poderá usar o "
```

Comment:
Did you mean to add this spacing?

Reply:
I inserted them just because I saw these same spaces in the original English sentences. Aren't them needed?